### ISSUE TITLE
Help reading a few words from a letter!

### ISSUE THREAD

**Issue by amberpeddicord** [amberpeddicord/amber-s_repo#1]
<img width="273" alt="mitfordletterquestion_10:2" src="https://user-images.githubusercontent.com/50875456/66088375-e68e4400-e548-11e9-88b0-e4378a588643.png">

<img width="183" alt="Screen Shot 2019-10-02 at 7 15 42 PM" src="https://user-images.githubusercontent.com/50875456/66088459-27865880-e549-11e9-8609-746fefa2b1eb.png">

@ebeshero 
I'm currently sorting photos, and I'm having trouble reading these few words! I'm trying to see if a few pages go together, but I can't tell what this is trying to say!

**Comment by ebeshero** [comment]
@amberpeddicord Hmm. In the top picture I see:  

```
actors
– If his passion 
```
(but I am not sure of that last word—does it fit the surrounding context?)

In the bottom picture I see

```
& sensibility 
```
but I suppose it could also be 

```
& feasibility
```

Does that help a little?

**Comment by amberpeddicord** [comment]
@ebeshero Yes! She was talking about an actor and how he could inspire an actress (I believe her name was Miss Phillips?) so that makes perfect sense. Thank you!

**Comment by ebeshero** [comment]
@amberpeddicord Yay! And yes, that actress would be Miss Philips (can't remember if that's one `l` or two in her name.
